docs/admin/kubeadm.md

```diff
@@ -260,6 +260,22 @@ These environment variables are a short-term solution, eventually they will be i
 
 If you want to use kubeadm with an http proxy, you may need to configure it to support http_proxy, https_proxy, or no_proxy.
 
+For example, your kube master node ip is 10.18.17.16, and you have a proxy support both http/https on 10.18.17.16 port 8080
```

Comment:
### Revision suggestion:

For example, if your kube master node IP address is 10.18.17.16 and you have proxy support both http/https on 10.18.17.16 port 8080, you can use the following command: